lib/minitest/hammer_coverage_plugin.rb

```diff
@@ -0,0 +1,26 @@
+require 'coverage_reporter'
+require "json"
+
+module Minitest
+  def self.plugin_hammer_coverage_options(opts, options)
+    opts.on "-c", "--coverage", "Generate coverage reports for API endpoints" do
+      options[:coverage] = true
+    end
+    opts.on "-v=VERSION", "--version=VERSION", "version" do |v|
+      options[:version] = v.split("=")[1]
+    end
+  end
+
+  def self.plugin_hammer_coverage_init(options)
+    Minitest.reporter.reporters.clear
+    data_dir = "test/data"
+    Minitest.after_run do
+      if options[:version]
+        unless Dir.entries(data_dir).include? options[:version]
+          raise StandardError.new "Version is not correct"
+        end
+      end
+        Minitest::CoverageRunner.new("#{data_dir}#{ options[:version] || ENV['TEST_API_VERSION'] || 2.1 }/foreman_api.json").run_tests
```

Comment:
I'd vote for having version check somewhere in one place for this case and [this case](https://github.com/theforeman/hammer-cli-foreman/blob/0aff7c637c85cb6d51a0cc6248249f4b2fca39c6/test/test_helper.rb#L20), so we don't need to change e.g. `2.1` into `2.2` in two places.